internal/core/src/common/Types.h

```diff
@@ -36,19 +39,28 @@ constexpr auto MAX_TIMESTAMP = std::numeric_limits<Timestamp>::max();
 
 using engine::DataType;
 using engine::idx_t;
+constexpr auto MAX_ROW_COUNT = std::numeric_limits<engine::idx_t>::max();
 
 using ScalarArray = proto::schema::ScalarField;
 using DataArray = proto::schema::FieldData;
 using VectorArray = proto::schema::VectorField;
 using IdArray = proto::schema::IDs;
 using MetricType = faiss::MetricType;
+using InsertData = proto::segcore::InsertRecord;
```

Comment:
why not use "InsertRecord" ?

Reply:
There is already a class nema InsertRecord